### ISSUE TITLE
TypeError on logOn. // at TCPConnection.send (/steam-user/components/connection_protocols/tcp.js)

### ISSUE THREAD

**Issue by dewythefirst** [DoctorMcKay/node-steam-user#392]
TypeError on logOn.
The app runs fine for days and weeks - it never happens any time soon after launch, so unfortunately, it wasn't something I managed to catch and debug myself.

I use proxy for steamuser - it works fine, shouldn't be an issue, but it might be an important detail.

**Version**
Module: 4.20.2
Node: 14.16.1

![image](https://user-images.githubusercontent.com/22560580/159120665-1f932a6d-50a9-4940-9b00-79958431ef56.png)

```
steam_1       |     TypeError: Cannot read property 'write' of undefined
steam_1       |     at TCPConnection.send (/app/node_modules/steam-user/components/connection_protocols/tcp.js:152:15)
steam_1       |     at SteamUser._send (/app/node_modules/steam-user/components/messages.js:478:19)
steam_1       |     at SteamUser._sendLogOn (/app/node_modules/steam-user/components/logon.js:249:7)
steam_1       |     at WebSocket.<anonymous> (/app/node_modules/steam-user/components/connection_protocols/websocket.js:125:14)
steam_1       |     at WebSocket.emit (events.js:327:22)
steam_1       |     at ClientRequest.<anonymous> (/app/node_modules/websocket13/lib/client.js:219:8)
steam_1       |     at ClientRequest.emit (events.js:315:20)
steam_1       |     at TLSSocket.socketOnData (_http_client.js:547:11)
steam_1       |     at TLSSocket.emit (events.js:315:20)
steam_1       |     at addChunk (internal/streams/readable.js:309:12)
steam_1       |     at readableAddChunk (internal/streams/readable.js:284:9)
steam_1       |     at TLSSocket.Readable.push (internal/streams/readable.js:223:10)
steam_1       |     at TLSWrap.onStreamRead (internal/stream_base_commons.js:188:23)
docker_steam_1 exited with code 1
```

**Comment by nolddor** [comment]
@dewythefirst is the issue still valid?

**Comment by dewythefirst** [comment]
Haven't seen it in quite a while.

**Comment by Sadzurami** [comment]
This happened with us too
![image](https://github.com/DoctorMcKay/node-steam-user/assets/47674424/34d982eb-7c61-4636-959a-3bc1a301cbc9)

**Comment by DoctorMcKay** [comment]
Fixed in #475
